docs/en/event-daemon/event-daemon.md

```diff
@@ -47,4 +56,49 @@ A plugin handles:
+
+## How event driven triggers work
+
+Below is a simple diagram of EventLogEntries generated by {% include product %}. Your Script will use the API to get a list of events that have occurred since the last time it asked. It will then look at each event type (e.g., {% include product %}_Task_Change) and see if any of them are ones that you care about.
+
+Once it finds an event that is interesting, it will examine the details of the event even further (e.g., what field was changed, what the value was changed to, etc. At this point you can even use the API to request more information for an entity if you need to).
+
+If the event proves worthy, the Script will then act on the event and execute whatever code you decide needs to be executed whether its using the {% include product %} API, or something in your pipeline, or both. When there are no more events to look at, it repeats the process and use the API to get a list of events that have occurred since the last time you asked.
+
+![Event log polling](./images/dv-writing-event-triggers-event-log-polling-01.png)
+
+## Polling the EventLog versus triggers
+
+{% include product %} provides a constant stream of event information and you can listen to it all and act only on the events you care about. This provides the following advantages over having {% include product %} control triggers itself:
+
+*   **Flexible** : Your trigger Scripts can run independent of {% include product %}. This allows your Script to interact both with {% include product %} and your pipeline in any way you want. You define the rules and actions as you wish without being bound by any constraints. {% include product %} doesn’t need to know anything about the event triggers you have. All it needs to do is keep generating the EventLogEntries. You control every other aspect of what happens next.
+*   **Remote** : Your Scripts can run from anywhere that has network access to the {% include product %} server. Your Script simply needs API access to run.
+*   **Multiplicity** : You can have multiple Scripts running concurrently. Different departments may have different needs and thus be listening for different events. There’s no restriction saying that all triggers be run from the same Script. You may wish to break up your triggers into separate logical Scripts. The polling query is very light and doesn’t have any noticeable impact on performance.
+*   **Accountability** : If your Scripts make changes to {% include product %}, they too create their own events, allowing you to see exactly what Scripts made changes.
+
+## Event types
+
+All internal event types follow the format **`{% include product %}_[entity_type]_[New|Change|Retirement]`**. Some examples are `{% include product %}_Shot_New` and `{% include product %}_Asset_Change`. For more information, see the [Event types documentation](https://github.com/shotgunsoftware/shotgunEvents/wiki/Technical_Overview#event-types).
```

Comment:
I assume the `include product` directive is going to resolve to `ShotGrid` at  some point. I don't think the event names are going to change though. So we won't have `ShotGrid_Shot_New` events. They will likely still be named `Shotgun_Shot_New`. I'm not sure who to ask, but this should be confirmed.

Reply:
For the sake of this documentation, let's match what it was. I'll fix those, thanks!